WordPress/Classes/ViewRelated/Domains/Register/RegisterDomainDetails/ViewController/RegisterDomainDetailsViewController+HeaderFooter.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+extension RegisterDomainDetailsViewController {
+
+    func configureTableFooterView() {
+        //Creating a UIView with a custom frame because table tableFooterView doesn't support autolayout
+        let footer = UIView(frame: CGRect(x: 0,
+                                          y: 0,
+                                          width: view.frame.size.width,
+                                          height: Constants.buttonContainerHeight))
+        footerView.frame = footer.frame
+        footer.addSubview(footerView)
+        footer.pinSubviewToAllEdges(footerView)
+        tableView.tableFooterView = footer
+    }
+
+    func privacyProtectionSectionHeader() -> RegisterDomainSectionHeaderView? {
+        return sectionHeader(title: Localized.PrivacySection.title,
+                             description: Localized.PrivacySection.description)
+    }
+
+    func contactInformationSectionHeader() -> RegisterDomainSectionHeaderView? {
+        return sectionHeader(title: Localized.ContactInformation.title,
+                             description: Localized.ContactInformation.description)
+    }
+
+    func sectionHeader(title: String, description: String) -> RegisterDomainSectionHeaderView? {
+        guard let view = tableView.dequeueReusableHeaderFooterView(
+            withIdentifier: RegisterDomainSectionHeaderView.identifier
+            ) as? RegisterDomainSectionHeaderView else {
+                return nil
+        }
+        view.setTitle(title)
+        view.setDescription(description)
+        return view
+    }
+
+    func privacyProtectionSectionFooter() -> EpilogueSectionHeaderFooter? {
+        guard let view = tableView.dequeueReusableHeaderFooterView(
+            withIdentifier: EpilogueSectionHeaderFooter.identifier
+            ) as? EpilogueSectionHeaderFooter else {
+                return nil
+        }
+        view.titleLabel?.attributedText = termsAndConditionsFooterTitle
+        view.titleLabel?.numberOfLines = 0
+        view.titleLabel?.lineBreakMode = .byWordWrapping
+        view.topConstraint.constant = 8
```

Comment:
Why is this 8? Can we make it a constant and give it a descriptive name maybe?

Reply:
sure, I defined a constant for this, it is 8 just because of the designs and no other reasons